src/common/snippets/include/snippets/lowered/pass/propagate_buffer_offset.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "pass.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+/**
+ * @interface PropagateBufferOffset
+ * @brief Get Buffer offset and set it to the ports of connected memory access operations.
+ *        Should be called after `SolveMemoryBuffer`.
+ * @ingroup snippets
+ */
+class PropagateBufferOffset: public RangedPass {
```

Comment:
This pass doesn't look like `RangedPass`: we anyway iterates through `buffer_expressions`.
```suggestion
class PropagateBufferOffset: public Pass {
```

Reply:
Totally agree! Thank you!